### PR TITLE
+act,rem,str add SECURITY marker to logs

### PR DESCRIPTION
**Pull request by ktoso**
Add a few Security markers to log statements

### CODE REVIEW

**Comment by patriknw** [comment]
One missing logging might be the things related to `PotentiallyHarmful` when untrusted mode is enabled.

**Comment by ktoso** [comment]
Good idea, I'll quickly skim for it

**Comment by ktoso** [comment]
Added markers for PossiblyHarmful things during untrusted mode

File: akka-actor/src/main/scala/akka/event/Logging.scala [hunk 1]
```diff
@@ -1678,3 +1680,49 @@ object NoLogging extends LoggingAdapter {
...
+  final override def warning(marker: LogMarker, template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = ()
+  final override def info(marker: LogMarker, message: String): Unit = ()
+  final override def info(marker: LogMarker, template: String, arg1: Any, arg2: Any): Unit = ()
+  final override def info(marker: LogMarker, template: String, arg1: Any, arg2: Any, arg3: Any): Unit = ()
+  final override def info(marker: LogMarker, template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = ()
+  final override def debug(marker: LogMarker, message: String): Unit = ()
+  final override def debug(marker: LogMarker, template: String, arg1: Any): Unit = ()
+  final override def debug(marker: LogMarker, template: String, arg1: Any, arg2: Any): Unit = ()
+  final override def debug(marker: LogMarker, template: String, arg1: Any, arg2: Any, arg3: Any): Unit = ()
+  final override def debug(marker: LogMarker, template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = ()
```
Comment (johanandren) [inline akka-actor/src/main/scala/akka/event/Logging.scala]: Good stuff :D

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Super weird compile failure... debugging

**Comment by ktoso** [comment]
Ok fixed the silly compile error

**Comment by ktoso** [comment]
Resolves https://github.com/akka/akka/issues/21670

**Pull request merged**